app/src/main/kotlin/com/wire/android/ui/common/snackbar/SwipeableSnackbar.kt

```diff
@@ -51,7 +43,8 @@ import kotlin.math.roundToInt
  * @see SnackbarData
  * @see SnackbarHostState
  */
-@OptIn(ExperimentalFoundationApi::class)
+
+// TODO uncomment all lines when anchoredDraggable will be available on new compose version
```

Comment:
suggestion: add the info about which particular version we would need to have to make it work again, maybe also mark that in the comment in `libs.versions.toml` next to the current version number, so that if someone updates it, will see that it can be now uncommented

I would also mark each one of the comments that are related to this, because with time we can have more things commented, or remove commented code completely and just add reference to this PR in comment so that it would be still easy to put back 😄